### PR TITLE
vcpu_cve: add case for cve status

### PR DESCRIPTION
**Pull request by dzhengfy**
This is to check guest cpu's CVE status.
Case ID: VIRT-294940

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
` (1/1) type_specific.io-github-autotest-libvirt.vcpu.cve.guest_cpu_cve_status.host_passthrough: WARN: Test passed but there were warnings during execution. Check the log for details. (72.18 s)`

libvirt-9.0.0-7.el9.x86_64

**Comment by dzhengfy** [comment]
@LuyaoHuang, please help review

**Review by Yingshun — APPROVED** [review]
LGTM

File: libvirt/tests/src/cpu/vcpu_cve.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import re
```
Comment (chunfuwen) [inline libvirt/tests/src/cpu/vcpu_cve.py:1]: what cve means here? is it one cpu sub feature name or is it something like bugzilla level we often mention. So maybe add one description in this python about what this python want to achieve.

**Comment by LuyaoHuang** [comment]
ACK from my point view

**Review by chloerh — APPROVED** [review]

File: libvirt/tests/src/cpu/vcpu_cve.py (hunk 1 above)
Comment (dzhengfy) [reply libvirt/tests/src/cpu/vcpu_cve.py:13]: Updated.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**